versions.txt

```diff
@@ -3,7 +3,5 @@
 
 google-cloud-datastore:2.2.0:2.2.1-SNAPSHOT
 google-cloud-datastore-bom:2.2.0:2.2.1-SNAPSHOT
-google-cloud-datastore-parent:2.2.0:2.2.1-SNAPSHOT
```

Comment:
@Neenu1995 can you take a look at why this was deleted?

Reply:
the parent version is same as the google-cloud-library version. Doesn't have to be tracked on versions.txt